tests: make aggregate_state_exception_memory_leak deterministic

azat opened the pull request:
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Sometimes if the 30 seconds is not enough for the query to fail (i.e. under [TSan](https://s3.amazonaws.com/clickhouse-test-reports/38748/d863f6ce1ece810c66ddf8bd89575825d3f2595f/stateless_tests__thread__actions__%5B2/3%5D.html)),
the error will be ignored and the test will fail.

Drop that loops away, since the query fails everytime and convert it to simple `.sql` test.

Follow-up for: #11496 (cc @alexey-milovidov )

alexey-milovidov commented:
But the whole point of this test was to check for a memory leak by triggering OOM by running the query in a loop.

azat committed:
tests: make aggregate_state_exception_memory_leak tests more deterministic

Sometimes if the 30 seconds is not enough for the query to fail (i.e.
under TSan [1]), the error will be ignored and the test will fail.

  [1]: https://s3.amazonaws.com/clickhouse-test-reports/38748/d863f6ce1ece810c66ddf8bd89575825d3f2595f/stateless_tests__thread__actions__%5B2/3%5D.html

Follow-up for: #11496 (cc @alexey-milovidov)
Signed-off-by: Azat Khuzhin <a.khuzhin@semrush.com>

azat commented:
>But the whole point of this test was to check for a memory leak by triggering OOM by running the query in a loop, and specifically the case when it throws an exception.

I see, thanks. Fixed the patch.

azat commented:
>[Stateless tests flaky check (address, actions) — Tests are not finished, fail: 0, passed: 54](https://s3.amazonaws.com/clickhouse-test-reports/38754/d1d717d4e841ec44c159724f5198d1a2c3e04efc/stateless_tests_flaky_check__address__actions_.html)

Timeout - OK

>[Stateless tests (release, s3 storage, actions) — fail: 1, passed: 4184, skipped: 57](https://s3.amazonaws.com/clickhouse-test-reports/38754/d1d717d4e841ec44c159724f5198d1a2c3e04efc/stateless_tests__release__s3_storage__actions_.html)

- `00933_ttl_replicated_zookeeper_long`

>[Stress test (debug, actions) — Fatal message in clickhouse-server.log (see fatal_messages.txt)](https://s3.amazonaws.com/clickhouse-test-reports/38754/d1d717d4e841ec44c159724f5198d1a2c3e04efc/stress_test__debug__actions_.html)

- #33531
